generators/entity-server/templates/src/test/kotlin/package/web/rest/EntityResourceIT.kt.ejs

```diff
@@ -1123,18 +1129,20 @@ class <%= entityClass %>ResourceIT <% if (databaseType === 'cassandra') { %>: Ab
                 <%_ } _%>
         <%_ } else { _%>
             val <%= otherEntityName %>: <%= otherEntityNameCapitalized %>
+            <%_ if (databaseType === 'sql') { _%>
             if (findAll(em, <%= otherEntityNameCapitalized %>::class.java).isEmpty()) {
                 <%= otherEntityName %> = <%= otherEntityNameCapitalized %>ResourceIT.create<% if (fieldStatus === 'UPDATED_') { %>Updated<% } %>Entity(<% if (databaseType === 'sql') { %>em<% } %>)
-                <%_ if (databaseType === 'sql') { _%>
                 em.persist(<%= otherEntityName %>)
                 em.flush()
-                <%_ } _%>
-                <%_ if (databaseType === 'mongodb') { _%>
-            <%= otherEntityName %>.id = "fixed-id-for-tests"
-                <%_ } _%>
             } else {
-                <%= otherEntityName %> = findAll(em, <%= otherEntityNameCapitalized %>::class.java).get(0)
+                <%= otherEntityName %> = findAll(em, <%= otherEntityNameCapitalized %>::class.java).get(0);
```

Comment:
`;` ?

Is it `TestUtil.` ??? 
Is this `java` class here?

Reply:
Nice catch. Will fix

Reply:
`;` removed.

So that was indeed a usage of `TestUtil`. I have changed it though to an extension function for `EntityManager` so it reads better and now pass `KClass` instead and do the conversion with the function. Similarly change `TestUtils.equalsVerifier` to accept a `KClass` parameter.